Awesome tests!  Got one error perhaps from a missing file 
<details>
(jwst) user@dhcp-10-134-208-105 tests % pytest *
============================================================================ test session starts ============================================================================
platform darwin -- Python 3.10.0, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /Users/user/es_programs/jtow, configfile: setup.cfg
plugins: asdf-2.10.0, jwst-1.4.3
collected 10 items                                                                                                                                                          

jtow_nrca1_custBias_None.py ..                                                                                                                                        [ 10%]
jtow_nrca1_custBias_cycleBias.py .                                                                                                                                    [ 20%]
jtow_nrca1_custBias_lineIntercept.py .                                                                                                                                [ 30%]
jtow_nrca1_custBias_selfBias.py .                                                                                                                                     [ 40%]
jtow_nrca5_custBias_None.py F                                                                                                                                         [ 50%]
jtow_nrca5_custBias_cycleBias.py .                                                                                                                                    [ 60%]
jtow_nrca5_custBias_lineIntercept.py .                                                                                                                                [ 70%]
jtow_nrca5_custBias_selfBias.py .                                                                                                                                     [ 80%]
test_jtow.py .

================================================================================= FAILURES ==================================================================================
___________________________________________________________________________ Test_JTOW.test_run_jw ___________________________________________________________________________

self = <tests.jtow_nrca5_custBias_None.Test_JTOW testMethod=test_run_jw>

    def setUp(self):
        """Set up test fixtures, if any."""
    
        self.examParamPath = cwd+'/jtow_test_results/jtow_nrca5_custBias_None/jtow_nrca5_custBias_None_params.yaml' #Path to YAML file
    
>       self.jtow_obj = jtow.jw(paramFile = self.examParamPath) #Assign YAML file to JTOW object

jtow_nrca5_custBias_None.py:76: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../jtow/jtow.py:101: in __init__
    self.get_parameters(paramFile,directParam=directParam)
../jtow/jtow.py:125: in get_parameters
    self.param = read_yaml(paramFile)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

filePath = '/Users/user/es_programs/jtow/tests/jtow_test_results/jtow_nrca5_custBias_None/jtow_nrca5_custBias_None_params.yaml'

    def read_yaml(filePath):
>       with open(filePath) as yamlFile:
E       FileNotFoundError: [Errno 2] No such file or directory: '/Users/user/es_programs/jtow/tests/jtow_test_results/jtow_nrca5_custBias_None/jtow_nrca5_custBias_None_params.yaml'

../jtow/jtow.py:60: FileNotFoundError
============================================================================= warnings summary ==============================================================================
../../../miniconda3/envs/jwst/lib/python3.10/site-packages/_pytest/config/__init__.py:1233
  /Users/user/miniconda3/envs/jwst/lib/python3.10/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: collect_ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

../../../miniconda3/envs/jwst/lib/python3.10/site-packages/astropy/version.py:24
  /Users/user/miniconda3/envs/jwst/lib/python3.10/site-packages/astropy/version.py:24: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.version import LooseVersion

tests/jtow_nrca1_custBias_None.py::Test_JTOW::test_run_jw
  /Users/user/miniconda3/envs/jwst/lib/python3.10/site-packages/asdf/entry_points.py:46: AsdfWarning: asdf.resource_mappings plugin from package jwst==1.4.3 failed to load:
  
  RuntimeError: Missing resources directory
    warnings.warn(

tests/jtow_nrca1_custBias_None.py::Test_JTOW::test_run_jw
  /Users/user/miniconda3/envs/jwst/lib/python3.10/site-packages/asdf/entry_points.py:46: AsdfWarning: asdf.extensions plugin from package jwst==1.4.3 failed to load:
  
  KeyError: 'Resource unavailable for URI: asdf://stsci.edu/jwst_pipeline/manifests/jwst_transforms-1.0.0'
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================================================================== short test summary info ==========================================================================
FAILED jtow_nrca5_custBias_None.py::Test_JTOW::test_run_jw - FileNotFoundError: [Errno 2] No such file or directory: '/Users/user/es_programs/jtow/tests/jtow_t...
================================================================= 1 failed, 9 passed, 4 warnings in 26.81s ==================================================================

</details>

--- a/jtow/jtow.py
+++ b/jtow/jtow.py
@@ -82,7 +82,7 @@ def log_output(TargetName):
     with open(pwd+'/stpipe-log.cfg'.format(TargetName), 'w') as conf:
         config_object.write(conf)
         
-    print("A configuration file {}_stpipe-log.cfg and a log output file {}_pipeline.log for the JWST Pipeline will be created in the working directory".format(TargetName, TargetName))
+    print("A configuration file stpipe-log.cfg and a log output file {}_pipeline.log for the JWST Pipeline will be created in the working directory".format(TargetName, TargetName))
 
 class jw(object):
     def __init__(self,paramFile=defaultParamPath,directParam=None):

--- a/tests/jtow_nrca5_custBias_None.py
+++ b/tests/jtow_nrca5_custBias_None.py
@@ -40,7 +40,7 @@ if (os.path.exists(rawdata_file) == False):
 #HDUList = fits.open(file)
 
 #image2D = HDUList[1].data[0][0] #Acting as Bias File
-jtow_Params_file = cwd+'/jtow_test_results/jtow_nrca1_custBias_None/jtow_nrca1_custBias_None_params.yaml' #File for the param file 
+jtow_Params_file = cwd+'/jtow_test_results/jtow_nrca5_custBias_None/jtow_nrca5_custBias_None_params.yaml' #File for the param file 
 if (os.path.exists(jtow_Params_file) == False):
     jtow_Params = {'rawFileSearch': rawdata_file,
                    'outputDir': cwd+'/jtow_test_results/jtow_nrca5_custBias_None/',

